engineering/workflow.md

```diff
@@ -54,31 +55,155 @@ The team conducts a Sprint Planning meeting for 60 minutes at the beginning of e
+
+The board is a GitHub Projects board that is populated with with work items during the teams Iteration Planning activity.
+
 The team's goal is to complete all items on the Sprint Board by the end of the Sprint.
 This is a team commitment - while one person may be assigned to a deliverable, we all commit to working together to get the work done.
 
 :::{admonition} The Sprint Board should...
 :class: tip
-- Have enough items to keep the team occupied for the week
+- Be driven by 2i2c's product roadmap
+- With the exception of support tickets, work items should be related back to Initiatives.
+- Have enough items to keep the team occupied for the two week iteration
 - Not have so many items that a team member gets overwhelmed
 - Under-estimate our team's total capacity, to provide room for unexpected work (e.g., support work)
-- Have a team member assigned to each item on the board
+- Have at least one team member assigned to each committed item on the board
 :::
 
-The Sprint Board is broken down into these columns:
+The Sprint Board is broken down into different columns that represent the team's delivery workflow. The team owns the design of this workflow and should change the workflow process that best suits their way of working to optimize for sustainable delivery.
+
+The current queues of work represented by the board are:
+
+- {guilabel}`Backlog` represents items that haven't been started and not in Sprint.
+- {guilabel}`Refined` represents a list of items that have been refined and are ready to be worked on. The items in this queue are not committed items.
+- {guilabel}`Committed` items haven't been started and are expected to be completed by end of the Sprint. These items are identified during the Iteration Planning activity.
+- {guilabel}`In Progress` represents a queue of items being *actively* worked on. Each item should have at least one owner.
+- {guilabel}`In Review/Blocked` contains items that need to be review before being marked as done or that cannot be completed without additional actions/support. A long list of items in this queue indicate deficiencies in the team's Refinement activity.
+- {guilabel}`Done` contains the list of items that are complete! These should be celebrated archived in the next Sprint Planning meeting.
+
+
+(meetings:retrospective)=
+### The Retrospective Meeting
+
+At the end of each sprint, the team holds a retrospective meeting to reflect and identify actions to improve the team's ways of working and delivery process. The retrospective is the process through which the team achieves continuous improvement for **all** their other processes.
+
+#### The Duration of the Retrospective
+
+The retrospective is one hour long and is usually held on the last day of each sprint.
+It is held at a time to maximise attendance from the engineering team.
```

Comment:
Are we now considering smaller retros?

Reply:
I've missed out on discussion I think, but this comment seems outdated so I'll mark it as resolved with the decrease from 60 to 45 minutes.